libraries/AP_HAL_ChibiOS/hwdef/iomcu_f103_8MHz/hwdef.dat

```diff
@@ -132,7 +132,7 @@ define DMA_RESERVE_SIZE 0
 define SOFTSIG_MAX_SIGNAL_TRANSITIONS 128
 
 define IOMCU_FW TRUE
-define NO_FASTBOOT
+define AP_FASTBOOT_ENABLED
```

Comment:
should be 0?

Reply:
worth working out why CI didn't catch this as a build error

Reply:
we need to use Tools/scripts/build_iofirmware.py in CI

Reply:
> should be 0?

Fixed.

Reply:
> we need to use Tools/scripts/build_iofirmware.py in CI

https://github.com/ArduPilot/ardupilot/pull/23060